test/integration/integration_test.go

```diff
@@ -606,17 +607,18 @@ func TestDockerfileBuild(t *testing.T) {
 		AsDockerfile: tempdir + string(os.PathSeparator) + "MyDockerfile",
 	}
 	expected := []string{
-		"FROM docker.io/centos/nodejs-8-centos7",
+		"(?m)^FROM docker.io/centos/nodejs-8-centos7",
 		"\"io.openshift.s2i.build.commit.date\"",
 		"\"io.openshift.s2i.build.commit.id\"",
 		"\"io.openshift.s2i.build.commit.ref\"",
 		"\"io.openshift.s2i.build.commit.message\"",
 		"\"io.openshift.s2i.build.source-location\"",
 		"\"io.openshift.s2i.build.image\"=\"docker.io/centos/nodejs-8-centos7\"",
 		"\"io.openshift.s2i.build.commit.author\"",
-		"COPY --chown=1001:0 upload/src /tmp/src",
-		"RUN /usr/libexec/s2i/assemble",
-		"CMD /usr/libexec/s2i/run",
+		"(?m)^COPY upload/src /tmp/src",
+		"(?m)^RUN chown -R 1001:0.* /tmp/src",
+		"(?m)^RUN /usr/libexec/s2i/assemble",
+		"(?m)^CMD /usr/libexec/s2i/run",
```

Comment:
Updated integration tests to use regex so that I could:

1. Ensure newlines are created
2. Make it easier to verify commands were created.